### PR TITLE
Fund tokens with faucet plugin - Closes #6138 

### LINKED ISSUE #6138: Add fundTokens action in faucet plugin
### Description

- Add fundTokens action in the faucet plugin
- Input should be
```
{
  address: 'binary address in hex format',
}
```
- Action should create a transaction using registered passphrase and send it to the running application

### Acceptance Criteria

- Add test to check if the getToken action create and send the token transfer transaction
- Add test to check if error is returned when input is invalid
- Add test to check if error is returned when passphrase is not set

### PR DESCRIPTION
**Pull request by mitsuaki-u**
### What was the problem?

This PR resolves #6138 

*Note: Tests will be created in separate issue after new `testing utils` branch is merged in.

### How was it solved?

- Added dependencies
- Added fund tokens action
- Added create transfer transaction method

### How was it tested?

Run unit test for faucet plugin

### CODE REVIEW

File: framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts [hunk 1]
```diff
@@ -12,11 +12,14 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { Transaction } from '@liskhq/lisk-chain';
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts:15]: Transaction is not needed in the `plugin`

File: framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts [hunk 2]
```diff
@@ -120,14 +136,95 @@ export class FaucetPlugin extends BasePlugin {
...
+			this._options,
+		) as FaucetPluginOptions;
 	}
 
 	// eslint-disable-next-line class-methods-use-this, @typescript-eslint/no-empty-function
 	public async unload(): Promise<void> {}
+
+	private async _createTransferTransaction(address: string): Promise<string> {
+		// ModuleID: 2 (Token), AssetID:0 (TransferAsset)
+		const transferAssetInfo = this.schemas.transactionsAssets.find(
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts:180]: you can create api client using channel, and create and send the transaction using the client

File: framework-plugins/lisk-framework-faucet-plugin/test/unit/plugin/fund.spec.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+			.calledWith('app:getAccount')
+			.mockResolvedValue('1a020801' as never);
+		when(channelMock.invoke)
+			.calledWith('app:getNodeInfo')
+			.mockResolvedValue({ networkIdentifier: defaultNetworkIdentifier } as never);
+		fundTokensAction = faucetPlugin.actions.fundTokens;
+		await faucetPlugin.load(channelMock);
+	});
+
+	it('should error if action input is invalid', async () => {
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/test/unit/plugin/fund.spec.ts:126]: maybe these test couldve been covered in functional tests?

File: framework-plugins/lisk-framework-faucet-plugin/package.json [hunk 4]
```diff
@@ -39,6 +39,8 @@
 		"prepublishOnly": "npm run lint && npm test && npm run build && npm run build:check"
 	},
 	"dependencies": {
+		"@liskhq/lisk-chain": "^0.2.0",
```
Comment (shuse2) [inline framework-plugins/lisk-framework-faucet-plugin/package.json:42]: with below comments, this will not be needed

**Review by shuse2 — CHANGES_REQUESTED** [review]

**Review by ishantiw — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**